webapp/javascript/ui/StatusMessage/index.tsx

```diff
@@ -26,7 +33,8 @@ export default function StatusMessage({ type, message }: StatusMessageProps) {
         [getClassnameForType()]: true,
       })}
     >
-      {message}
+      <div>{message}</div>
+      <div className={styles.rightSideWrapper}>{rightSide}</div>
```

Comment:
Nit, but as you pointed b4, let's hide this div if `rightSide` is not available. Less stuff in the DOM is always a good idea :P

Reply:
is `:empty` rule suitable for this case?
`
.rightSideWrapper {
  &:empty {
    display: none;
  }
}
`